src/cache/DbQueryResultCache.ts

```diff
@@ -28,7 +28,8 @@ export class DbQueryResultCache implements QueryResultCache {
     constructor(protected connection: Connection) {
 
         const options = <SqlServerConnectionOptions|PostgresConnectionOptions>this.connection.driver.options;
-        this.queryResultCacheTable = this.connection.driver.buildTableName("query-result-cache", options.schema, options.database);
+        const cacheOptions = typeof this.connection.options.cache === "object" ? this.connection.options.cache : {};
+        this.queryResultCacheTable = this.connection.driver.buildTableName(cacheOptions.cacheTableName || "query_result_cache" , options.schema, options.database);
```

Comment:
Previous default table name was "query-result-cache" not "query_result_cache". Please be careful. 

*My personal preference (optional):
Maybe expose this part `cacheOptions.cacheTableName || "query_result_cache"` as separate variable. For me it is more readable in cases long lines.*

Reply:
Hi @vlapo 
Noted with thanks.
I have changed as you requested.
Unfortunately, I couldn't add a new feature in file CHAGELOG.md since I don't have access to add it.
Please lemme know if there is anything else I could.